route/build/src/kwt_route.f90

```diff
@@ -861,7 +853,8 @@ SUBROUTINE QEXMUL_RCH(IENS,JRCH,T0,T1,&            ! input
  ! ----------------------------------------------------------------------------------------
  IF(INIT) THEN
   INIT=.FALSE.
-  NULLIFY(USFLOW,NEW_WAVE,QD_TEMP,TD_TEMP)
+  !NULLIFY(USFLOW,NEW_WAVE,QD_TEMP,TD_TEMP)
+  !deallocate(USFLOW,NEW_WAVE,QD_TEMP,TD_TEMP)
```

Comment:
remove commented out code

Reply:
This section was removed since these variables are deallocated in the end of this subroutine